example/ExampleMaterialDesignControls/Pages/MaterialEntryPage.xaml.cs

```diff
@@ -11,18 +11,18 @@ public MaterialEntryPage()
 
             BindingContext = new MaterialEntryViewModel { DisplayAlert = DisplayAlert, Navigation = Navigation };
 
-            txtName.Focused += TxtName_Focused;
-            txtName.Unfocused += TxtName_Unfocused;
+            //txtName.Focused += TxtName_Focused;
```

Comment:
Can we add another MaterialEntry or add to a existing one in order to leave an example of using the focused and unfocused features?